budget-app/budget.py

```diff
@@ -0,0 +1,113 @@
+class Category:
+
+    def __init__(self, category):
+        self.ledger = []
+        self.amount = 0
+        self.name = category
+
+    def deposit(self, amount, desc=""):
+        self.ledger.append({"amount": amount, "description": desc})
+        self.amount += amount
+
+    def withdraw(self, amount, desc=""):
+
+        if self.check_funds(amount):
+            self.amount -= amount
+            self.ledger.append({"amount": -amount, "description": desc})
+            return True
+
+        return False
+
+    def get_balance(self):
+
+        return self.amount
+
+    def check_funds(self, amount):
+
+        return False if self.amount < amount else True
```

Comment:
It should be `return not (self.amount < amount)` ..... no need of extra booleans